T/Telescope/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Telescope"
+version = v"0.1.2"
+
+# Collection of sources required to build this package
+sources = [
+    ArchiveSource("https://github.com/jhigginbotham64/Telescope/archive/refs/tags/v$(version).tar.gz",
+              "025a24a5dfe4c4d5910ae0cad7664ec2c2fc21cce8ffaae56cc6b42300249de6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CFLAGS="-I${prefix}/include"
+export CXXFLAGS="-I${prefix}/include"
```

Comment:
Nitpick: please use the most specific [environment variable](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables), in this case `includedir`
```suggestion
export CFLAGS="-I${includedir}"
export CXXFLAGS="-I${includedir}"
```